.gitattributes

```diff
@@ -1,4 +1,17 @@
-bin/ export-ignore
-artifacts/ export-ignore
-package.json export-ignore
-gulpfile.js export-ignore
+# Auto-detect text files, ensure they use LF.
+* text=auto eol=lf
+
+# Exclude non-essential files from dist
+/bin export-ignore
+/artifacts export-ignore
+/tests export-ignore
+/.coveralls.yml  export-ignore
```

Comment:
You can use `/.* export-ignore` for all `dot` files.

Reply:
thanks, for your feedback.
i'm not sure that all `dot` (future) files  must be excluded.
IMHO: it's more safely careful exclude. Whereas if you do not exclude a file it's not a problem, but if you do it it could become a problem.